packages/discovery-provider/src/tasks/entity_manager/entities/developer_app.py

```diff
@@ -229,6 +267,43 @@ def validate_developer_app_tx(params: ManageEntityParameters, metadata):
             raise IndexingValidationError(
                 "Invalid Create Developer App Transaction, user has too many developer apps"
             )
+    elif params.action == Action.UPDATE:
+        if not address:
+            raise IndexingValidationError(
+                f"Invalid {params.action} Developer App Transaction, address is required and was not provided"
+            )
+
+        if address not in params.existing_records["DeveloperApp"]:
```

Comment:
Would be nice if we could share the validation for all of the params between the two transaction types?

Reply:
will take a stab at making it a bit nicer!